docs/learn/integrations/dagster.md

```diff
@@ -0,0 +1,113 @@
+You getmetadata store resources, an IOManager that moves `Increment`s between assets, and helpers to turn features into Dagster assets.
+
+## Installation
+
+```bash
+pip install "metaxy[dagster]"
+```
+
+## Quick Start
+
+Minimal root + downstream wiring:
+
+```python
+import dagster as dg
+import metaxy as mx
+import metaxy.ext.dagster as mxd
+
+store_resource = mxd.MetaxyMetadataStoreResource.from_config(store_name="dev")
+io_manager = mxd.MetaxyIOManager.from_store(store_resource)
+
+
+class RawFeature(
+    mx.BaseFeature,
+    spec=mx.FeatureSpec(key="demo/raw", id_columns=["id"], fields=["value"]),
+):
+    id: str
+    value: str
+
+
+class CleanFeature(
+    mx.BaseFeature,
+    spec=mx.FeatureSpec(
+        key="demo/clean",
+        deps=[mx.FeatureDep(feature="demo/raw")],
+        id_columns=["id"],
+        fields=["clean_value"],
+    ),
+):
+    id: str
+    clean_value: str
+
+
+@dg.asset(spec=mxd.build_asset_spec_from_feature(RawFeature))
+def raw_feature(store: mx.MetadataStore) -> mx.FeatureSpec:
+    samples = generate_samples()  # includes metaxy_provenance_by_field
+    with store:
+        inc = store.resolve_update(RawFeature, samples=samples)
+        if len(inc.added) > 0:
+            store.write_metadata(RawFeature, inc.added)
+    return RawFeature.spec()
+
+
+@dg.asset(
+    spec=mxd.build_asset_spec_from_feature(CleanFeature),
+    ins=mxd.build_asset_in_from_feature(RawFeature),
+)
+def clean_feature(diff: mx.Increment, store: mx.MetadataStore) -> mx.FeatureSpec:
+    if len(diff.added) > 0:
+        store.write_metadata(CleanFeature, transform(diff.added))
+    return CleanFeature.spec()
+
+
+defs = dg.Definitions(
+    assets=[raw_feature, clean_feature],
+    resources={"store": store_resource, "io_manager": io_manager},
+)
+```
+
+## Core Components
+
+- [`MetaxyMetadataStoreResource`][metaxy.ext.dagster.MetaxyMetadataStoreResource]: load configured stores (with optional fallbacks).
+  See class docstring for options.
+- [`MetaxyIOManager`][metaxy.ext.dagster.MetaxyIOManager]: moves `Increment`s between assets; supports partition, sample, and key filtering.
+  See the docstring for flags (`partition_key_column`, `sample_limit`, `sample_filter_expr`, `target_key_column`/`target_keys`).
+- Helpers:
+  - [`build_asset_spec_from_feature`][metaxy.ext.dagster.build_asset_spec_from_feature]: turns a feature into an `AssetSpec`.
+  - [`build_asset_in_from_feature`][metaxy.ext.dagster.build_asset_in_from_feature]: creates typed `AssetIn` mapping (optionally overriding which feature to resolve).
+
+## Execution patterns
+
+Most code lives in the runnable examples and helper docstrings. Pick a pattern and start from the example file:
+
+- Minimal: smallest end-to-end wiring — [examples/example-integration-dagster/minimal.py](https://github.com/anam-org/metaxy/blob/main/examples/example-integration-dagster/src/example_integration_dagster/minimal.py)
```

Comment:
I do not like these links: I could not get relative links to work as mkdocs only seems to work in the docs folder; these external links point to GH but only to main/latest not to a specific version - and we are currently also not released.



Is there a better way?



How can we handle references to examples properly?